osu.Game/Localisation/Mods/TargetPracticeModStrings.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation.Mods
+{
+    public static class TargetPracticeModStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.Mods.TargetPracticeMod";
+
+        /// <summary>
+        /// "Seed"
+        /// </summary>
+        public static LocalisableString Seed => new TranslatableString(getKey(@"seed"), "Seed");
+
+        /// <summary>
+        /// "Use a custom seed instead of a random one"
+        /// </summary>
+        public static LocalisableString SeedDescription => new TranslatableString(getKey(@"seed_description"), "Use a custom seed instead of a random one");
+
+        /// <summary>
+        /// "Practice keeping up with the beat of the song."
+        /// </summary>
+        public static LocalisableString Description => new TranslatableString(getKey(@"target_practice_description"), "Practice keeping up with the beat of the song.");
```

Comment:
I really don't think each mod's description should be in a separate localisation file. I'd either see:

- All descriptions in one file, all settings related strings in one file
- or as above, but separated by category. probably not a good idea because we may move mods between categories though.

Reply:
Maybe for mods that are not abstract or don't inherit from an abstract mod, I should do an `Orphan` category? So that we don't have to deal with mods moving categories because they don't belong.